cmd/meroxa/root/resources/create.go

```diff
@@ -233,3 +269,25 @@ func (c *Create) hasCredentials() bool {
 		c.flags.Token != "" ||
 		c.flags.SSL
 }
+
+func (c *Create) handlePrivateKeyFlags(ctx context.Context) error {
+	path := c.flags.PrivateKeyFile
+	if path != "" && c.flags.SSHPrivateKey == "" {
+		bytes, err := os.ReadFile(path)
+		if err != nil {
+			return fmt.Errorf("could not find SSH private key at %q."+
+				" Try a different path`", path)
```

Comment:
Could this one be omitted?

```suggestion
				" Try a different path", path)
```